examples/policies/host/lock-down-ingress.yaml

```diff
@@ -18,5 +18,5 @@ spec:
         protocol: TCP
       - port: "4240"
         protocol: TCP
-      - port: "8472"
+      - port: "REMOVE_ME_AFTER_DOUBLE_CHECKING_PORTS"
```

Comment:
Was this intending to replace the 22 port instead of 8472?

Reply:
Do we want to drop port 22? I thought that was one of the few pieces that didn't completely brick the nodes in these cases where incorrect policy was applied; at least someone can still SSH into the node & unload BPF filters if necessary.

Reply:
Maybe it's a good idea not to remove any of the existing ports, but instead add one extra that can be removed? I'm not sure what the implications are for if we drop 8472 whether this makes it more likely that users will deploy policies that might block tunnel traffic.

Reply:
this might have been accidentally removed?

Reply:
What if we replace this policy with the following?
```yaml
apiVersion: "cilium.io/v2"
kind: CiliumClusterwideNetworkPolicy
description: "Allow a minimum set of required ports on ingress of worker nodes"
metadata:
  name: "lock-down-ingress-worker-node"
spec:
  nodeSelector:
    matchLabels:
      type: worker
  ingress:
  - fromEntities:
    - world
  - toPorts:
    - ports:
      - port: "6443"
        protocol: TCP
      - port: "2379"
        protocol: TCP
      - port: "4240"
        protocol: TCP
      - port: "8472"
        protocol: UDP
      - port: "PLEASE_ADD_PORTS_RELEVANT_TO_YOUR_ENV"
        protocol: ANY
```
That would ensure external access is never broken (unless bug) while forcing the user to take a look and add relevant ports.

Reply:
that's what I've been trying to say since the beginning :cry:. However, the `fromEntities` does not seem correct in this case. The nodes won't be able to connect to kube-apiserver since the policy is _only_ allowing `world` and nodes are not `world`

Reply:
sorry for my misunderstanding @aanm. I think that's a great approach. for the `fromEntities`, do you think we should replace that with `remote-node` or `cluster`?

Reply:
> sorry for my misunderstanding @aanm. I think that's a great approach. for the `fromEntities`, do you think we should replace that with `remote-node` or `cluster`?

@jedsalazar I think it shouldn't be either. mostly because new nodes won't have an identity assigned, and therefore none of the _entity_ will work (well _world_ will probably work but I believe that's not our goal)